tests/test_util.py

```diff
@@ -49,17 +49,27 @@ def test_gatorgrader_home_is_set():
     )
 
 
-def test_gatorgrader_home_is_set_after_os_dictionary_set():
+def test_gatorgrader_home_is_set_after_os_dictionary_set(tmpdir):
     """Ensure that the GATORGRADER_HOME environment variable is set."""
-    os.environ["GATORGRADER_HOME"] = "/home/gkapfham/working/source/gatorgrader/"
+    tmpdir.mkdir("gatorgrader")
+    os.environ["GATORGRADER_HOME"] = str(tmpdir) + "/" + "gatorgrader/"
     gatorgrader_home = util.get_gatorgrader_home()
     assert gatorgrader_home is not None
     assert "gatorgrader" in gatorgrader_home
 
 
-def test_gatorgrader_home_is_set_after_os_dictionary_set_no_trailing():
+def test_gatorgrader_home_is_set_after_os_dictionary_set_no_trailing(tmpdir):
```

Comment:
I doubt the need for trailing/no-trailing slash test cases -- this is normally handled by either the OS itself or the low-level Python file interfaces.

Reply:
Yes, this is probably true. With that said, I don't think that this is a big issue and thus I am going to leave the test case for now. 

Importantly, I am going to have to delete a huge number of tests in the new version of the tool that adopts the linter interface and uses plugins.

I anticipate that I can remove this test, as needed, along with the removal of many other tests.